packages/tools/u-boot/package.mk

```diff
@@ -1,19 +1,20 @@
 ################################################################################
-#      This file is part of OpenELEC - http://www.openelec.tv
+#      This file is part of LibreELEC - https://www.libreelec.tv
```

Comment:
`https://www.libreelec.tv` -> `https://libreelec.tv`

Reply:
And Thanks again.